packages/xstate-react/CHANGELOG.md

```diff
@@ -2,12 +2,31 @@
 
 All notable changes to this project will be documented in this file.
 
+## [1.0.0-rc.7]
+
+- The `machine` passed into `useMachine(machine)` can now be passed in lazily:
+
+  ```js
+  const [state, send] = useMachine(() => createMachine(/* ... */));
+
+  // ...
+  ```
+
+  This has the benefit of avoiding unnecessary machine initializations whenever the component rerenders, without having to use `useMemo(...)`.
```

Comment:
IMHO mentioning useMemo here is not good because `useMemo` shouldn't be used for this. We only care about the very first machine received by us - all others are just ignored. `useMemo` could only help with avoiding [this warning](https://github.com/davidkpiano/xstate/blob/d8f765586b82bf191727d463b6b05019716d0b55/packages/xstate-react/src/useMachine.ts#L120-L123) but even then it wouldn't avoid it entirely as `useMemo` doesn't hold a semantic guarantee. Therefore `useMemo` was never a good tool for the job here.